src/python/pants/backend/debian/target_types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.goals.package import OutputPathField
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    DictStringToStringField,
+    SpecialCasedDependencies,
+    StringField,
+    Target,
+)
+from pants.util.docutil import doc_url
+
+
+class DebianControlFile(StringField):
+    alias = "control"
+    help = "Path to a Debian control file for the package to be produced."
```

Comment:
This should probably be the `sources` field, which will give you lots of important benefits like:

- already supports relative path syntax, where `sources=['foo.control']` really means `<path/to/BUILD>/foo.control`
- means file arguments will work, like `./pants package path/to/foo.control`
- lots of Rule API code already

You probably want to require it to be 1 file, and override the help message also:

https://github.com/pantsbuild/pants/blob/cec93fe9280fdea212eceaad03429f23ec8c312f/src/python/pants/backend/docker/target_types.py#L7-L10

Is there a sensible default? If not, this should have `required = True`.

Reply:
Ah that's useful, thank you. There is no sensible default, I am afraid. Shall we use this?

```
class DebianControlFile(Sources):
    required = True
    expected_num_files = 1
    help = "Path to a Debian control file for the package to be produced."
```